.updatecli/bump-opbeans-node.yml

```diff
@@ -0,0 +1,65 @@
+---
+## Workflow to periodically check if there is an available newer APM agent version, e.g.
+## "1.2.3". If so, then update to it.
+name: Bump elastic-apm-node to latest version
+
+scms:
+  githubConfig:
+    kind: github
+    spec:
+      user: '{{ requiredEnv "GIT_USER" }}'
+      email: '{{ requiredEnv "GIT_EMAIL" }}'
+      owner: elastic
+      repository: opbeans-node
+      token: '{{ requiredEnv "GITHUB_TOKEN" }}'
+      username: '{{ requiredEnv "GIT_USER" }}'
+      branch: main
+
+actions:
+  opbeans-node:
+    kind: github/pullrequest
+    scmid: githubConfig
+    spec:
+      automerge: false
```

Comment:
Would `automerge: true` be the desired state?

I set it to false for now because the repository does not allow it